pkg/plan/recipe/install_plans.go

```diff
@@ -269,15 +269,29 @@ func BuildCNIPlan(cni string, manifests [][]byte) plan.Resource {
 //BuildSealedSecretPlan creates a sub-plan to install sealed secrets so we can check secrets into GitHub for GitOps
 func BuildSealedSecretPlan(sealedSecretVersion, ns string, manifest []byte) plan.Resource {
 	b := plan.NewBuilder()
+	fileCRD, err := manifests.Manifests.Open("05_sealed_secret_crd.yaml")
+	if err != nil {
+		log.Fatalf("%v", err)
+	}
+	manifestbytesCRD, err := ioutil.ReadAll(fileCRD)
+	if err != nil {
+		log.Fatalf("%v", err)
+	}
+
+	b.AddResource("install:sealed-secret-crd",
+		&resource.KubectlApply{Manifest: manifestbytesCRD, Filename: object.String("SealedSecretCRD.yaml"),
+			WaitCondition: "condition=Established"})
+
 	b.AddResource("install:sealed-secrets-key", &resource.KubectlApply{Manifest: manifest})
-	file, err := manifests.Manifests.Open("05_sealed_secret_controller.yaml")
+	file, err := manifests.Manifests.Open("06_sealed_secret_controller.yaml")
```

Comment:
I'm curious why we needed to update the file name from `05` to `06`

Reply:
I just added as 05 to the CRD and thought to increase the controller